pl_bolts/models/rl/common/networks.py

```diff
@@ -335,3 +336,81 @@ def forward(self, input_x: Tensor) -> Tensor:
         noisy_weights = self.sigma_weight * self.epsilon_weight.data + self.weight
 
         return F.linear(input_x, noisy_weights, bias)
+
+
+class ActorCategorical(nn.Module):
+    """
+    Policy network, for discrete action spaces, which returns a distribution
+    and an action given an observation
+    """
+
+    def __init__(self, actor_net: nn.Module):
```

Comment:
Let's add `-> None` for `__init__` as stated in [PEP 484](https://www.python.org/dev/peps/pep-0484/#the-meaning-of-annotations):
> Note that the return type of __init__ ought to be annotated with -> None. 

```suggestion
    def __init__(self, actor_net: nn.Module) -> None:
```